Makefile

```diff
@@ -10,7 +10,7 @@ GO ?= go
 PKG_CARBONAPI=github.com/go-graphite/carbonapi/cmd/carbonapi
 PKG_CARBONZIPPER=github.com/go-graphite/carbonzipper/cmd/carbonzipper
 
-carbonapi:
+carbonapi: $(shell find . -name '*.go' | grep -v 'vendor')
```

Comment:
Not sure what's this change is about

Reply:
It declares all non-vendored `*.go` files in the project to be prerequisites for the `carbonapi` target (see [the make manual]) This means that if any Go file in the project changes, then writing `make carbonapi` will rebuild the binary. If we omit the prerequisites, `make carbonapi` will not rebuild the binary if it already exists, even if the existing binary was built from source files that have now changed. For the `carbonapi` and `carbonzipper` targets, this is handy for checking they still compile when making new changes.

[the make manual]: https://www.gnu.org/software/make/manual/make.html#Rule-Example